app/src/main/java/com/example/firstandroid/MainActivity.kt

```diff
@@ -39,25 +19,17 @@ class MainActivity : ComponentActivity() {
         lifecycleScope.launch {
```

Comment:
I still don't think we need lifecycle scope in here, but not huge. I have a feeling it isn't doing anything